recipe/build.sh

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env bash
 
 # find out toolchain C++ standard
-CXX_STANDARD=14
+CXX_STANDARD=17
 if [[ ${CXXFLAGS} == *"-std=c++14"* ]]; then
-    echo "14"
-    CXX_STANDARD=14
+    echo "14 -> 17"
+    export CXXFLAGS="${CXXFLAGS} -std=c++17"
+    if [[ ! -z ${MACOSX_DEPLOYMENT_TARGET} ]]; then
+        export MACOSX_DEPLOYMENT_TARGET=10.13
```

Comment:
Why this?

Reply:
`-std=c++17`: to append behind the `-std=c++14` flag. CMake will not set extra flags if the [compiler default](https://gist.github.com/ax3l/53db9fa8a4f4c21ecc5c4100c0d93c94) is already C++17 and I request it.

`MACOSX_DEPLOYMENT_TARGET`: I thought to have seen that the default set here is smaller in the env, but I need `10.13`/`10.14` for C++17 support in the STDlib.

Reply:
You don't need 10.13/10.14. can you revert that?

Reply:
Sure!